subsys/bluetooth/host/Kconfig

```diff
@@ -173,19 +173,23 @@ config BT_SETTINGS_USE_PRINTK
 endif # BT_SETTINGS
 
 config BT_WHITELIST
-	bool "Enable whitelist support"
+	bool "DEPRECATED: Use BT_FA_LIST instead"
```

Comment:
Is this a new policy? I cannot remember seeing deprecation of Kconfig options.

Reply:
We've done a mix of both in the past, I'm taking the full path here.